framework/tests/base_test.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from browser.py_quality_services import PyQualityServices
+from framework.utils.settings_data import SettingsData
+
+
+@pytest.fixture(scope="session", autouse=True)
+def prepare_browser_factory(request):
```

Comment:
better to rename this fixture to be just "browser".
Also fixtures must be defined in conftest, not in base_test